src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -330,6 +337,7 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setNormalizer('choices_as_values', $choicesAsValuesNormalizer);
 
         $resolver->setAllowedTypes('choices', array('null', 'array', '\Traversable'));
+        $resolver->setAllowedTypes('empty_choices', array('null', 'string'));
```

Comment:
On line 314 default for `'choices' => array(),` but line 339 allowed types is set to `array('null', 'array', '\Traversable')`.

Is `null` really needed ?